lib/commands/new.js

```diff
@@ -9,6 +10,8 @@ const isValidProjectName = require('../utilities/valid-project-name');
 const normalizeBlueprint = require('../utilities/normalize-blueprint-option');
 const mergeBlueprintOptions = require('../utilities/merge-blueprint-options');
 
+const rmdir = util.promisify(fs.remove);
```

Comment:
Since `fs-extra` supports promises directly, you can remove this line and then update callsites of `rmdir` to use `fs.remove` directly.